packages/keyring-controller/src/KeyringController.test.ts

```diff
@@ -614,6 +624,17 @@ describe('KeyringController', () => {
                 },
               );
             });
+          !cacheEncryptionKey &&
+            it('should not set encryptionKey and encryptionSalt in state', async () => {
+              await withController(
+                { skipVaultCreation: false, cacheEncryptionKey },
+                async ({ controller }) => {
+                  await controller.createNewVaultAndKeychain(password);
```

Comment:
Nit: Typically we like to have a newline between each "phase" of a test, e.g. between the thing-under-test and the assertions about it

```suggestion
                  await controller.createNewVaultAndKeychain(password);

```

See more on that here: https://github.com/MetaMask/contributor-docs/blob/main/docs/unit-testing.md#-highlight-the-exercise-phase